src/modules/vendor/components/courses/ProfileFollowUp.vue

```diff
@@ -181,6 +180,17 @@ export default {
       const type = this.messageTypeOptions.find(type => type.value === value);
       return type ? type.label : '';
     },
+    courseNotStartedYet () {
+      const slots = this.course.slots.filter(slot => this.$moment().isAfter(slot.endDate))
+      return !slots.length;
+    },
+    loadMessageTypeOptions () {
+      if (this.courseNotStartedYet()) this.messageTypeOptions.push({ label: 'Convocation', value: CONVOCATION });
+      this.messageTypeOptions.push({ label: 'Rappel', value: REMINDER });
+    },
+    setDefaultMessageType () {
+      this.messageType = this.messageTypeOptions[0] ? this.messageTypeOptions[0].value : REMINDER;
```

Comment:
Je ne suis pas sure de comprendre la condition. 
Je mettrais plutot `this.courseNotStartedYet ? CONVOCATION : REMINDER`, non ?

Reply:
oui